usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -94,15 +98,13 @@ def handle(self, *args, **options):
         loader = ThreadedDataLoader(Procurement, field_map=field_map, value_map=value_map)
         loader.load_from_file(options['file'][0])
 
-    def get_agency(self, row):
-        agency = Agency.objects.filter(subtier_code=self.get_agency_code(row['maj_agency_cat'])).first()
+    def get_agency(self, agency_string):
+        agency = Agency.objects.filter(subtier_code=self.get_agency_code(agency_string)).first()
```

Comment:
I see this pattern of `first()` in our ETL a lot. At some point, do you think it makes sense to do a `get()` instead so that we can capture MultipleObject errors and log if we multiple records in a reference table? Or are we ok relying on unique indexes at the db level to protect us from these types of integrity issues?